data/categories.txt

```diff
@@ -1584,7 +1584,7 @@ nl:^Kledingwinkel|4kleding
 fi:^Vaatekauppa
 fr:^Boutique de vêtements|4vêtements
 de:^Bekleidungsgeschäft|Kleidung|Laden
-hu:^Ruhabolt|4ruhák
+hu:^Ruházati bolt|3ruhák
```

Comment:
Maybe move the "3" to the first synonym?

Reply:
​OK, it can be so.​